src/ResourceMover/docs/Add-AzResourceMoverMoveResource.md

```diff
@@ -20,7 +20,7 @@ Add-AzResourceMoverMoveResource -MoveCollectionName <String> -Name <String> -Res
 ```
 
 ## DESCRIPTION
-Creates or updates a Move Resource in the move collection.
+Creates or updates a Move Resource in the move collection.<br>The 'Add-AzResourceMoverMoveResource' command remains same for both 'RegionToRegion' and 'RegionToZone' type move collections.
```

Comment:
Hi @yashjain4 , change description in docs/*.MD will be ignored. Please change description by changing the description of swagger operation in directive. Example:

```
 - from: swagger-document


	where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/backupVaults/{vaultName}/backupInstances/{backupInstanceName}/findRestorableTimeRanges"].post


	transform: $["description"] = "Finds the valid recovery point in time ranges for the restore."
```